test/omega/test.lean

```diff
@@ -326,5 +326,8 @@ example (a b : Int) (h : a > 7) (w : b > 2) : a > 0 ↔ b > 0 := by omega
 -- Verify that we can prove implications:
 example (a : Int) : a > 0 → a > -1 := by omega
 
+-- Verify that we can handle double negation:
+example (x y : Int) : x + 1 ≤ y → ¬ y + 1 ≤ x := by omega
```

Comment:
why is this a double negation example? Shouldn't `false_or_by_contra` end up just introducing the hypotheses?

Reply:
No, `false_or_by_contra` doesn't do any intros. Probably that should change!

Reply:
Done.